src/content/_index.en.md

```diff
@@ -36,7 +36,7 @@ Distribution documentation (aka man pages) is [available here](https://www.ovn.o
 We have weekly IRC meetings you can participate in:
 
 [OVN weekly](http://eavesdrop.openstack.org/meetings/ovn_community_development_discussion/2021/)
-IRC meeting every Thu 1715 UTC in **#openvswitch** on [irc.libera.chat](https://libera.chat) 
+IRC meeting every Thu 09:15 AM US Pacific in **#openvswitch** on [irc.libera.chat](https://libera.chat)
```

Comment:
If only the world could agree to abolish DST, this would work.

Both `UTC` and `US Pacific` are unfortunately ambiguous in this context for anyone not local to the TZ in question. 

If I pop pacific into www.timeanddate.com/worldclock/converter.html for exampe, I get to choose between the DST and non-DST version, which may or may not be obvious to the person reading this.

Are there other choices here, put Boston/New York in parenthesis for example? (I use New York as TZ for this in my personal calendar)

Reply:
> If only the world could agree to abolish DST, this would work.
> 

:)

> Both `UTC` and `US Pacific` are unfortunately ambiguous in this context for anyone not local to the TZ in question.
> 

I copied the #openvswitch IRC channel banner but we can probably improve it indeed.

> If I pop pacific into [www.timeanddate.com/worldclock/converter.html](http://www.timeanddate.com/worldclock/converter.html) for exampe, I get to choose between the DST and non-DST version, which may or may not be obvious to the person reading this.
> 

That's a bit unfortunate UI, I think.  DST and non-DST are "clearly" defined (errrm... DST is never clear but whatever).  E.g., on the wikipedia [Pacific Time Zone page](https://en.wikipedia.org/wiki/Pacific_Time_Zone):

```
In the United States and Canada, this time zone is generically called the Pacific Time Zone.
Specifically, time in this zone is referred to as Pacific Standard Time (PST) when standard time
is being observed (early November to mid-March), and Pacific Daylight Time (PDT) when daylight
saving time (mid-March to early November) is being observed. In Mexico, the corresponding time
zone is known as the Zona Noroeste (Northwest Zone) and observes the same daylight saving
schedule as the United States and Canada. The largest city in the Pacific Time Zone is Los Angeles,
whose metropolitan area is also the largest in the time zone.
```

> Are there other choices here, put Boston/New York in parenthesis for example? (I use New York as TZ for this in my personal calendar)

I see a few options:
a. Link to the current time in the PT zone on timeanddate.com, e.g.: https://www.timeanddate.com/worldclock/converter.html?iso=20230728T080000&p1=tz_pt
b. Choose [Los Angeles, California](https://www.timeanddate.com/worldclock/usa/los-angeles) as the time reference as it is the "largest in the time zone" and mention it as example for "Pacific Time".
c. Use UTC for both the IRC banner and here on the website.

Wdyt, @fnordahl?

Reply:
Apologies for the tardy response @dceara.

I guess the main problem is that US/Europe has chosen different times of year for when DST is enacted, AFAIR there is a two week window between when US and Europe does the switch.

So for anyone reading the page the meaning of DST is relative to their location.

I think b) is the most appropriate choice as whoever reads the page can then use their favorite choice of time zone conversion tool to arrive at the correct time for the meeting.